recipes/python-geotiepoints/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "python-geotiepoints" %}
+{% set version = "1.1.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7fe57a4e504c24116ce25d88122d35629fd0973d409fd62bafd6a9f80f8a0694" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```

Comment:
I'd recommend adding cython in here so that we regenerate the c binding to match out numpy versions cleanly